Xamarin.Forms.Platform.iOS/Renderers/EditorRenderer.cs

```diff
@@ -39,7 +39,10 @@ protected override void OnElementChanged(ElementChangedEventArgs<Editor> e)
 
 			base.OnElementChanged(e);
 
-			CreatePlaceholderLabel();
+			if (e.NewElement != null)
```

Comment:
Maybe add this check above where it creates the label as well? or just check if the label is not null?